template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/raz/RangerRazDatalakeConfigProviderTest.java

```diff
@@ -132,7 +134,8 @@ static Object[][] razCloudPlatformAndCmVersionDataProvider() {
         return new Object[][]{
                 // testCaseName cloudPlatform runtime
                 {"CloudPlatform.AWS", CloudPlatform.AWS, "7.2.2"},
-                {"CloudPlatform.AZURE", CloudPlatform.AZURE, "7.2.1"}
+                {"CloudPlatform.AZURE", CloudPlatform.AZURE, "7.2.2"},
```

Comment:
version mismatch

Reply:
yes. it has been done as part of this change.